movies/src/components/addMovie/AddMovie.js

```diff
@@ -1,19 +1,40 @@
 import React, { useState } from 'react';
 import PropTypes from 'prop-types';
+import {bindActionCreators} from "redux";
+import {connect} from "react-redux";
 import Input from '../common/input/Input';
 import Button from '../common/button/Button';
 import MultiSelectInput from '../common/multiSelectInput/MultiSelectInput';
 import Datepicker from '../common/datepicker/Datepicker';
 import CloseButton from '../closeButton/CloseButton';
 import MovieActionPopup from '../movieActionPopup/MovieActionPopup';
 import Constants from '../constants';
+import {editMoviesList} from "../../store/actions/actions";
 
-export default function AddMovie(props) {
+const matchDispatchToProps = (dispatch) => {
```

Comment:
`const matchDispatchToProps = { updateMovies: editMoviesList }`
would be enough here
OR
you could use `useDispatch` hook from `react-redux`